cdap-ui/pom.xml

```diff
@@ -272,12 +275,22 @@
                     <copy todir = "${stage.opt.dir}/dist">
                       <fileset dir = "dist"/>
                     </copy>
+                    <copy todir = "${stage.opt.dir}/bin">
+                      <fileset dir = "node">
+                        <include name="node" />
+                      </fileset>
+                    </copy>
                     <copy todir = "${stage.opt.dir}/node_modules">
                       <fileset dir = "node_modules"/>
                     </copy>
                     <copy todir = "${stage.opt.dir}/templates">
                       <fileset dir = "templates"/>
                     </copy>
+                    <copy todir = "${stage.opt.dir}">
```

Comment:
Never mind, I thought you had the slash outside the quotes. It doesn't matter, either way. I can add it, for consistency, but it's redundant and really should be removed from the other locations, instead.